src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -73,7 +73,7 @@ class Microsoft_Launcher : public PowertoyModuleIface
     static const int MAX_WAIT_MILLISEC = 10000;
 
     // Hotkey to invoke the module
-    Hotkey m_hotkey = { .key = 0 };
+    Hotkey m_hotkey = { .key = {} };
```

Comment:
this change won't affect how the initialization was performed, since before the fields of `m_hotkey` were properly initialized as well.

We can check that it was already worked as expected like this:

![image](https://user-images.githubusercontent.com/1828123/104183733-7648c080-5423-11eb-8785-ca056ebc0cad.png)

Reply:
My bad, I meant Hotkey m_hotkey = { };

Reply:
that would be same thing as well

Reply:
It's not the same thing as `Hotkey m_hotkey = { .key = {} }`

Reply:
what would be the difference?

Reply:
`m_hotkey = { };` resets all the properties, `Hotkey m_hotkey = { .key = 0 }` resets `.key`.

Reply:
But I agree that this is not the cause of the bug, so I'll close this PR.

Reply:
> m_hotkey = { }; resets all the properties, Hotkey m_hotkey = { .key = 0 } resets .key.

you can see from the screenshot attached that it's doing the same.